runtime/build.rs

```diff
@@ -276,7 +276,7 @@ fn main() {
     let snapshot_slice = &[];
     #[allow(clippy::needless_borrow)]
     #[allow(clippy::disallowed_methods)]
-    std::fs::write(&runtime_snapshot_path, snapshot_slice).unwrap();
+    std::fs::write(runtime_snapshot_path.clone(), snapshot_slice).unwrap();
```

Comment:
Why is this being cloned now? We probably need to change the clippy allow to a rust allow above instead.

Reply:
> I don't think this needs to be cloned now. We probably need to change the clippy allow to a rust allow above instead.

Done.